Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/hud.yml

```diff
@@ -0,0 +1,15 @@
+- type: entity
+  parent: ClothingEyesHudMedical
+  id: ClothingEyesHudSyndicateMedical # This actually does nothing at the moment, but Wizden are close/have/are about to get the MedHud working again.
+  name: syndicate medical visor
+  description: The syndicate's professional head-up display, designed for better detection of humanoids, their health for their subsequent elimination.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/synd.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/synd.rsi
+  - type: ShowSyndicateIcons
```

Comment:
![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/25ec00e4-86af-4550-ba8b-1bfc1ce0d97b)
I have an issue here with the Hud having ShowSyndicateIcons. Namely that since it reveals certain antagonist roles to the wearer, it opens up the syndi paramed to abuse by powergamers. There's nothing really stopping security from mugging the poor guy for his HUD, and then use it to go play "Spot the Antags".